### ISSUE TITLE
cannot find symbol class ......._

### ISSUE THREAD

**Issue by hyoungtaikkim** [androidannotations/androidannotations#2183]
This is a template, please adjust it accordingly to your specific issue.<br/>
You may also want to read our [issue guide](https://github.com/androidannotations/androidannotations/wiki/HowToWriteAnIssue).

**AndroidAnnotations version**: 4.5.2  (tried 3.3.2 and 4.4.0 too)

**Android compile SDK version**: 27

**Annotated code**:
```java
your code
```

**Expected generated code**:
```java
expected generated code
```

**Actual generated code**:
```java
actual generated code
```

**Stacktrace in case of AA crash**:

Hi,

I know this topic has something to do with #2093 , but as some people in there have indicated, it still doesn't work even with your suggestion in there. My symptom is exactly same as what other people complained especially people's complain after your comment. 

I am using Android Studio 3.0.1, Gradle version 4.4, Android Plugin Version 3.1.4 and I don't want to use apt and I need to use JavaVersion.VERSION_1_8 (this must have caused big change though.).

I really appreciate your excellent work in any case.

Thanks.

**Comment by dodgex** [comment]
can you provide a small project to reproduce the problem?

**Comment by hyoungtaikkim** [comment]
Hi,

Maybe it might have something to do with Android Studio 3.0.1 and the above versions. I mean I had many other weird problems too like "Module cannot find sdk" and etc. and may statements in the code became red with "cannot find symbol or class" or whatever messages. So I tried to use Android Studio 3.1.4 instead with same gradle files and all those weird problem have gone mysteriously. Anyway, during this work, I have removed all the Android Annotations anyway.

Thanks.
